packages/networkaccess/src/index.ts

```diff
@@ -0,0 +1,27 @@
+import { OnRpcRequestHandler } from '@metamask/snaps-types';
+import { panel, text } from '@metamask/snaps-ui';
+
+// eslint-disable-next-line jsdoc/require-jsdoc
+async function getPage() {
+  const response = await fetch(
+    'https://metamask.github.io/test-snaps/index.html',
+  );
+  return response.text();
+}
+
+export const onRpcRequest: OnRpcRequestHandler = ({ request }) => {
+  switch (request.method) {
+    case 'networkAccessTest':
+      return getPage().then((page) => {
+        return snap.request({
+          method: 'snap_dialog',
+          params: {
+            type: 'alert',
+            content: panel([text(`${page}`)]),
```

Comment:
This seems like it would be very difficult to it on screen? Is it just a huge scrollbar?

Reply:
Yea, you can probably choose a different example or figure out something else, this one looks weird a little bit. Maybe a web page with less content, some simple JSON response from our API, etc...

![Screenshot 2023-05-11 at 17 01 16](https://github.com/MetaMask/test-snaps/assets/13301024/7112fb54-c6fa-48ee-997d-7e11bb51e459)

Reply:
Maybe you can use snaps registry JSON response `https://acl.execution.metamask.io/latest/registry.json` if there are no objections to that.
It looks decent:
![Screenshot 2023-05-11 at 17 29 44](https://github.com/MetaMask/test-snaps/assets/13301024/de8d62f9-49d0-4aab-a55c-61f6ceccf2e3)

Reply:
resolved in latest commit

Reply:
I do not know that this file will remain static, as I know the other file would. This could be solved by a static file or reply that was deployed somewhere permanently. If this is static then you can disregard this comment.

Reply:
Sorry for using a big ugly file before, it was just an unchanging file on metamask.io and, for testing purposes, doesn't really matter what it looks like, and was verifiable in E2E - which this other suggested file is as well - but i am afraid it would eventually change, breaking the E2E test verifying contents.

Reply:
You're right if you're verifying that file, then my suggestion is not the best solution.

I was thinking maybe other people will use this page to (try) test snap capabilities and it would be better if something looks more user friendly. Also, fetching and processing less content will theoretically make test lighter or faster.
It would probably be the best to add some fixed JSON file within this repository, something that can be statically hosted within this test-snaps website, but I'm not sure at the moment if something like that is possible with this setup 🤔.

Reply:
Ok, here is what you can do:

1. Create folder called `static` inside the root of the `site` directory just alongside the files like `gatsby-config.ts` (not in `src`).
2. Then in that (`static`) folder create a file named `test-data.json`.
3. Add some contents in that file like: 
```json
{
  "result": "FETCHED_SUCCESSFULLY"
}
```
4. Run `yarn build`.

Now you will have `test-data.json` file within the `public` directory and you can access it (after running `yarn start`) on `http://localhost:8080/test-data.json`.

This way you'll have test data that is in your control and is optimal in many ways for testing.
But, you will first need to create another PR, get it merged and published, so you can use the URL or if you're not testing it on this PR (CI), you can predict the URL and add it in code (because localhost URL will not work on CI or website after publishing). 

![Screenshot 2023-05-12 at 12 30 18](https://github.com/MetaMask/test-snaps/assets/13301024/39adb2f4-1b33-48de-93e3-fd716fcafa34)

Reply:
The problem is that I need a file that is deployed to a website, such as the test-snaps site on GitHub (MetaMask.GitHub.io) because GitHub actions and CircleCI don't access the files from localist during testing...

Reply:
It will be accessible once you deploy it.
For example here https://metamask.github.io/test-snaps/latest/test-data.json, once file is added to a build, PR merged, public files deployed.
Just like this icon for example that is accessible now: https://metamask.github.io/test-snaps/latest/icons/icon-48x48.png
If it works on localhost, it will work in deployed version if it's a static file added during the build process.

Anyway, this is only my suggestion, I'm not requesting a change if you prefer another approach.

Reply:
I'm going to address that static file in a seperate PR. (it is also added here now.)

Reply:
https://github.com/MetaMask/test-snaps/pull/171

Reply:
I don't see a reason to do that in a separate PR since you've also made the change here.